React/Views/ScrollView/RCTScrollContentView.m

```diff
@@ -14,13 +14,20 @@
 
 @implementation RCTScrollContentView
 
+- (void)didMoveToWindow
+{
+  [super didMoveToWindow];
+  RCTScrollView *scrollView = (RCTScrollView *)self.superview.superview;
+  [scrollView updateContentSizeIfNeeded];
+}
+
 - (void)reactSetFrame:(CGRect)frame
 {
   [super reactSetFrame:frame];
 
   RCTScrollView *scrollView = (RCTScrollView *)self.superview.superview;
 
-  if (!scrollView) {
+  if (!scrollView || !self.window) {
```

Comment:
curious why this change is necessary - my understanding is that the bug happens due to a race of when we have a window after the initial call to this method happens.

if that's the case, then shouldn't we be good after we re-update the content size in didMoveToWindow? why do subsequent calls to reactSetFrame fail?